docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
```

Comment:
Is the Compose file required if we are already mentioning `services` in the CI file?